how-to/add_controller.rst

```diff
@@ -1,119 +1,120 @@
 
-    The **cloud region** is the region in which the controller is being bootstrapped. 
+MicroK8s Controller
+-------------------
 
-    The **DNS name** is the full DNS name that will be given to the controller, often it is wise to make the host name the same as the controller name within a particular domain. 
+Bootstrap a new Juju controller in MicroK8s. We will name this controller ``workload-microk8s`` as it will be running our workloads
+as opposed to our original controller which only deploys JAAS.
 
-    The **Candid URL** is the URL of the candid server that is providing the centralised identity service for the JAAS system. 
+.. code:: bash
 
-    The **JIMM URL** is the URL of the JIMM system providing the JAAS service.
+    juju bootstrap microk8s workload-microk8s --config login-token-refresh-url=http://jimm-endpoints.jimm.svc.cluster.local:8080/.well-known/jwks.json
 
-    +----------------------+----------------------+
-    | Parameter            | Environment variable |
-    +======================+======================+
-    | Controller name      | $NAME                |
-    +----------------------+----------------------+
-    | Cloud                | $CLOUD               |
-    +----------------------+----------------------+
-    | Cloud Region         | $REGION              |
-    +----------------------+----------------------+
-    | DNS name             | $DNS                 |
-    +----------------------+----------------------+
-    | Candid URL           | $CANDID              |
-    +----------------------+----------------------+
-    | JIMM URL             | $JIMM                |
-    +----------------------+----------------------+
+Once this process is complete we will switch back to JIMM and add the controller to JIMM.
 
+.. code:: bash
 
-2. Now we are ready to bootstrap a controller. Please note the constraints here are the ones used for production JAAS services and should be suitable for most loads. If it is anticipated that the JAAS system will have a different model profile then we encourage you to determine the appropriate constraints for your system: 
+    juju switch jimm
+    jimmctl controller-info workload-microk8s ~/snap/jimmctl/common/k8s-controller-info.yaml --local --tls-hostname juju-apiserver
+    jimmctl add-controller ~/snap/jimmctl/common/k8s-controller-info.yaml
 
-    ``juju bootstrap --bootstrap-constraints="root-disk=50G cores=8 mem=8G" --config login-token-refresh-url=https://$JIMM/.well-known/jwks.json --config allow-model-access=true --config public-dns-address=$DNS:443 $CLOUD/$REGION $NAME``
+The above commands create a YAML file with information about the controller and passes this information to JIMM.
 
-3. Next the controller should be put into HA mode (optional): 
+.. note::
 
-    ``juju enable-ha``
+    A Juju server's default certificate contains a `SAN <https://en.wikipedia.org/wiki/Subject_Alternative_Name>`__ for the name ``juju-apiserver``.
```

Comment:
suggestion to end with: which is why we specify the ``--tls-hostname juju-apiserver`` when running the ``controller-info`` command.